index.md

```diff
@@ -9,24 +10,32 @@ toc: false
 
 ## Overview 
 
-GraphWalker is a [Model-Based testing](https://en.wikipedia.org/wiki/Model-based_testing) tool. It reads models in the shape of [directed graphs](https://en.wikipedia.org/wiki/Directed_graph), and generate [test] paths from these graphs.
+GraphWalker is a [Model-Based testing](https://en.wikipedia.org/wiki/Model-based_testing) tool. 
```

Comment:
This is not really a change, is it? You added some new lines, but the mark down rendering is still the same, right?
If so, I suggest not to make a change here.